AUTHORS

```diff
@@ -1,4 +1,5 @@
-# This is the official list of Go Cloud authors for copyright purposes.
+# This is the official list of Go Cloud Development Kit authors for copyright
```

Comment:
If you're doing this, then you need to change the copyright line across the repo to match:

```
Copyright 2019 The Go Cloud Development Kit Authors
```

Reply:
Yup, this is done already.

Reply:
Year does not need to be updated.